src/lib/OpenEXR/ImfStandardAttributes.h

```diff
@@ -140,24 +155,310 @@ IMF_STD_ATTRIBUTE_DEF (longitude, Longitude, float)
+// independent mapping from a unit of tint to a distance on a chromaticity
+// diagram. One camera vendor might choose a color space (e.g. the
+// CIE 1960 UCS) and have a unit amount of tint represent some delta uv
+// distance from the point by the cammeraCCTSetting and a tint value of 0.
+// Another might choose to express the effect of tint by analogy to a
+// traditional unit from a film workflow, e.g. a Kodak or Rosco color
+// correction filter. About the only guaranteed commonality is that all
+// camera vendor tint schemes have positive values shift the adopted
+// scene white towards green, and negative values toward magenta.
+//
+// If the camera vendor maps cameraCCTSetting to a point defined by
+// a linear crossfade between a Planckian blackbody locus and loci of
+// CIE D Series illuminants, the slope of the tint isotherm at the
+// exact points where the linear crossfade starts and ends can be 
+// indeterminate and an inverse mapping from chromaticity to a pair
+// of CCT and tint can be one-to-many.
+//
+// The most common use for the cameraTintSetting attribute is to feed its
+// value into a camera-vendor-provided application or API, along with
+// a cameraCCTSetting attribute value, to reproduce the color processing
+// done in-camera on set.
+//
+
+IMF_STD_ATTRIBUTE_DEF (cameraTintSetting, CameraTintSetting, float)
+
+//
+// cameraColorBalance -- chromaticity in CIE 1960 UCS coordinates
+// indicating a color the user of the camera would like the camera
+// to treat as neutral, and corresponding to a particular camera
+// configuration of make, model, camera firmware version, CCT setting
+// and tint setting.
+//
+// Note that this is not necessarily (or even probably) the same
+// chromaticity as that of the scene adopted white stored in an
+// adoptedNeutral attribute (if present).
+//
+// For example, if a physical digital cinema camera was configured with
+// a CCT of 3200K and a tint of -3 (in some camera vendor dependent unit),
+/// and the camera output had been processed such that the image containing
+// this attribute was encoded as per SMPTE ST 2065-4:2023, then 
+// the adoptedNeutral attribute would have the value corresponding to
+// the ACES neutral chromaticity, very near that of CIE Illuminant D60,
+// whereas the cameraColorBalance would have a chromaticity much, much
+// warmer than that of the adoptedNeutral attribute.
+// 
+
+IMF_STD_ATTRIBUTE_DEF (cameraColorBalance, CameraColorBalance, IMATH_NAMESPACE::V2f)
+
 //
 // isoSpeed -- the ISO speed of the film or image sensor
 // that was used to record the image
 //
 
 IMF_STD_ATTRIBUTE_DEF (isoSpeed, IsoSpeed, float)
 
+//
 // exposure -- exposure time, in seconds
+//
 
 IMF_STD_ATTRIBUTE_DEF (expTime, ExpTime, float)
 
 //
-// aperture -- the camera's lens aperture, in f-stops (focal length
-// of the lens divided by the diameter of the iris opening)
+// shutterAngle -- shutter angle, in degrees
```

Comment:
What is the expected value when there is no shutter? Would it be zero? 360? I suppose zero so the default is trivial?

Reply:
Now about not present if there is no meaningful shutter to have an angle?

Reply:
If the camera has no shutter, then the file header should not have a shutterAngle attribute. If it were necessary for some processing reason for it to have one, then it would have a shutter angle of 360º, corresponding to 'always open'. Should I add a note to that effect?

Reply:
Presumably all these attributes apply to CG rendered images. An image rendered with no motion blur would presumably have a shutterAngle of 0 instead of 360?

Reply:
I think it would be grand to capture all of these suggestions.

~ not present if there's no meaningful data to record
~ always open should be encoded as 360
~ an instantaneous image such as a CG rendered image with no motion blur could indicate a shutterAngle of zero

Reply:
per tsc meeting, understood to mean "forensic" information as recorded on set.